com.github.gmg137.netease-cloud-music-gtk.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.github.gmg137.netease-cloud-music-gtk
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: netease-cloud-music-gtk
+rename-icon: netease-cloud-music-gtk
+rename-desktop-file: netease-cloud-music-gtk.desktop
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
```

Comment:
I don't see wayland permssions.

Reply:
So the `fallback-x11` should always used with `wayland`?

Reply:
the name say "fallback". If the app supports wayland, then `--socket=wayland` to display on wayland and `--socket=fallback-x11` if wayland is not available and the app support a fallback to x11. (I no know app that don't support x11).

If the app doesn't support wayland then only `--socket=x11`

Reply:
Added.